sys/include/bitfield.h

```diff
@@ -60,7 +64,7 @@ static inline void bf_set(uint8_t field[], size_t idx)
  */
 static inline void bf_unset(uint8_t field[], size_t idx)
 {
-    field[idx / 8] &= ~(1u << (idx % 8));
+    field[idx / 8] &= ~(1u << (7 - (idx % 8)));
```

Comment:
I wanted to suggest

```suggestion
    field[idx >> 3] &= ~(1u << (~idx & 0x7));
```

but turns out it generates [the same code](https://godbolt.org/z/hy429u).

Reply:
If the compiler couldn't figure out such arithmetic optimizations, I would be seriously worried about the optimizer ;-).